src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  line-height: 21.86px;
```

Comment:
The `line-height` property typically does not require a unit unless it's relative to the font size, in which case 'em' or unitless values are commonly used. For example, `line-height: 1.35;`.